WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -85,15 +83,37 @@ final class BlogDashboardViewController: UIViewController {
 extension BlogDashboardViewController {
 
     private func createLayout() -> UICollectionViewLayout {
-        let layout = UICollectionViewCompositionalLayout {
+        UICollectionViewCompositionalLayout {
             (sectionIndex: Int, layoutEnvironment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection? in
 
-            return self.createQuickLinksSection()
+            if sectionIndex == 0 {
+                return self.createQuickActionsLayoutSection()
+            } else {
+                return self.createDefaultLayoutSection()
+            }
         }
-        return layout
     }
 
-    private func createQuickLinksSection() -> NSCollectionLayoutSection {
+    private func createQuickActionsLayoutSection() -> NSCollectionLayoutSection {
```

Comment:
Can you add a comment on this method stating what it does? It is because of the scrolling behavior, right?